pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -545,12 +559,14 @@ func isOnCluster(publishingStrategyIngress *cloudingressv1alpha1.ApplicationIngr
 		return false
 	}
 	if publishingStrategyIngress.RouteSelector.MatchLabels != nil {
```

Comment:
Should this be an or instead of nested?

Reply:
I agree with @fahlmant on this one. Can this be re-worked to avoid the nested conditionals?